core/tests/test_utils.py

```diff
@@ -1613,37 +1506,21 @@ def save_new_exp_with_states_schema_v21(self, exp_id, user_id, title):
             title: str. The title of the exploration.
         """
         exp_model = exp_models.ExplorationModel(
-            id=exp_id,
-            category='category',
-            title=title,
-            objective='Old objective',
-            language_code='en',
-            tags=[],
-            blurb='',
-            author_notes='',
-            states_schema_version=21,
+            id=exp_id, category='category', title=title,
+            objective='Old objective', language_code='en', tags=[], blurb='',
+            author_notes='', states_schema_version=21,
             init_state_name=feconf.DEFAULT_INIT_STATE_NAME,
-            states=self.VERSION_21_STATE_DICT,
-            param_specs={},
-            param_changes=[]
-        )
+            states=self.VERSION_21_STATE_DICT, param_specs={}, param_changes=[])
         rights_manager.create_new_exploration_rights(exp_id, user_id)
 
         commit_message = 'New exploration created with title \'%s\'.' % title
         exp_model.commit(
-            user_id, commit_message, [{
-                'cmd': 'create_new',
-                'title': 'title',
-                'category': 'category',
-            }])
+            user_id, commit_message,
+            [{'cmd': 'create_new', 'title': 'title', 'category': 'category'}])
```

Comment:
Where there is more than one field in a dict, please do use one key-value pair per line.

(As mentioned above, it may be simplest to just do this always.)